### PR TITLE
doc: make the release snippet easier to use

### PR DESCRIPTION
**Pull request by alexeagle**
Right now we tell everyone to copy-paste a thing which assumes they have typescript

### CODE REVIEW

**Comment by gregmagolan** [comment]
How about just removing the "bins" entirely from the snippet instead?

**Comment by gregmagolan** [comment]
Although, your diff made me think of adding this,
````
# OPTIONAL: reference your .npmrc file if you have one in your repository to pick-up registry & auth configurations
npmrc = "//.npmrc"
```

**Comment by alexeagle** [comment]
I think the `bins` entry here is load-bearing for the e2e to work though

**Comment by alexeagle** [comment]
added .npmrc, good idea

File: e2e/workspace/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,3 @@
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
```
Comment (gregmagolan) [inline e2e/workspace/BUILD.bazel:3]: should also add this to `ci.yaml` so it doesn't bit rot & remote the `---SNIP---` from the old location

**Review by gregmagolan — APPROVED** [review]

File: e2e/workspace/BUILD.bazel (hunk 1 above)
Comment (gregmagolan) [reply e2e/workspace/BUILD.bazel:3]: will do in follow-up

**Pull request merged**